cmd/minikube/cmd/root.go

```diff
@@ -98,6 +99,17 @@ func Execute() {
 		} else {
 			os.Args = append([]string{RootCmd.Use, callingCmd, "--"}, os.Args[1:]...)
 		}
+	} else if filepath.Ext(callingCmd) == ".exe" && driver.IsMicrosoftWSL() {
+		var found = false
+		for _, a := range os.Args {
+			if a == "--force" {
+				found = true
+				break
+			}
+		}
+		if !found {
+			exit.Message(reason.WrongBinary, "Cannot run Windows binary inside WSL, please download linux binary from https://minikube.sigs.k8s.io/docs/start/. Or you can use '--force' to force execution which would be at your own risk.")
```

Comment:
You are trying to run windows .exe binary inside WSL, for better integration please use Linux binary instead. otherwise if you still want to do this, you can do it using --force"

Reply:
done